hack/jenkins/e2e.sh

```diff
@@ -53,6 +54,7 @@ else
       export MASTER_SIZE=${MASTER_SIZE:-"t2.small"}
     else
       export MASTER_SIZE=${MASTER_SIZE:-"g1-small"}
```

Comment:
Totally agree.  We should never be running on shared cores for CI e2e tests.  It introduces unnecessary timing variability, which makes debugging more difficult.